io.defn.Franz.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.defn.Franz
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: Franz
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
```

Comment:
Why do you need full home?

Reply:
To store metadata in a way that the user can move around ([stored](https://github.com/Bogdanp/Franz/blob/master/core/appdata.rkt#L20-L24) in `$XDG_CONFIG_HOME/io.defn.Franz`).

Reply:
XDG_CONFIG_HOME in the sandbox is set, it will go to `~/.var/app/<app-id>/config`, I don't think `~/.config` will be used. If that's the only reason you can remove it.